tools/bootstrap-patches/0001-Don-t-assume-option-is-always-a-string-closes-2896.patch

```diff
@@ -0,0 +1,36 @@
+From 9a9686e1722f0ee534a014d176edd8b2c2aceb06 Mon Sep 17 00:00:00 2001
+From: Carson <cpsievert1@gmail.com>
+Date: Fri, 15 May 2020 10:49:43 -0500
+Subject: [PATCH] Don't assume option is always a string, closes #2896
+
+Fixes a Bootstrap bug introduced by https://github.com/twbs/bootstrap/pull/16011
+---
+ inst/www/shared/bootstrap/js/bootstrap.js     |   2 +-
+ inst/www/shared/bootstrap/js/bootstrap.min.js | Bin 39680 -> 39702 bytes
+ 2 files changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/inst/www/shared/bootstrap/js/bootstrap.js b/inst/www/shared/bootstrap/js/bootstrap.js
+index 170bd608..6202bc2f 100644
+--- a/inst/www/shared/bootstrap/js/bootstrap.js
++++ b/inst/www/shared/bootstrap/js/bootstrap.js
+@@ -720,7 +720,7 @@ if (typeof jQuery === 'undefined') {
+       var data    = $this.data('bs.collapse')
+       var options = $.extend({}, Collapse.DEFAULTS, $this.data(), typeof option == 'object' && option)
+ 
+-      if (!data && options.toggle && /show|hide/.test(option)) options.toggle = false
++      if (!data && options.toggle && typeof option === 'string' && /show|hide/.test(option)) options.toggle = false
+       if (!data) $this.data('bs.collapse', (data = new Collapse(this, options)))
+       if (typeof option == 'string') data[option]()
+     })
+diff --git a/inst/www/shared/bootstrap/js/bootstrap.min.js b/inst/www/shared/bootstrap/js/bootstrap.min.js
```

Comment:
Best not to include the patch for the minified part, since applying the patch will fail in the future.